xmppserver/src/main/webapp/error-serverdown.jsp

```diff
@@ -31,14 +32,14 @@
 <body>
 
 <div id="jive-header">
-<table cellpadding="0" cellspacing="0" width="100%" border="0">
+<table>
 <tbody>
     <tr>
      <td>
-         <img src="<%= path %>/<%= AdminConsole.getLogoImage() %>" border="0" alt="<%= AdminConsole.getAppName() %> <fmt:message key="error.serverdown.admin_console" />">
+         <img src="<%= path %>/<%= AdminConsole.getLogoImage() %>" alt="<%= AdminConsole.getAppName() %> <fmt:message key="error.serverdown.admin_console" />">
```

Comment:
`getLogoImage()` appears to default to `/images/header-title.gif`, but this image appears missing in my docker container.
I can't find it in the repo, but possibly I'm being daft, or it's renamed in the build, or it's just an access path for a different file on disk :man_shrugging:

Reply:
wow... last touched in 2006 - and I never even saw this. I don't think anyone sees that JSP, ever?

Apparently, this is taken from `admin-sidebar.xml`, which has references to old wildfire stuff. This was already failing prior to this PR, by the way.

I have removed the references from `admin-sidebar.xml`, and added an `if` statement to this usage, which will prevent the image from being (attempted to be) displayed, when the reference is not there.

Modifications made in #ff7e2d1211264d7c442a401ba0cf2cec81b9d236